src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -121,4 +121,9 @@ public override object Clone()
 
         return result;
     }
+
+    public float BaseSpeed()
```

Comment:
Use a property

Reply:
The value of BaseSpeed relies on the organelles, which are a mutable property. That leaves us with the choices of refactoring organelle layout to not allow modification without modifying speed, trusting that all future devs will remember you can't change one without the other, or making some redundant calculations for the dependent value.

Reply:
I meant that literally convert this calculation to a get only property.
These kinds of methods that just compute or retrieve one value should be made as properties in C# and not methods.
I guess I've been coding so much C# lately that I didn't even consider that my comment could be misinterpreted...